Assets/MixedRealityToolkit.Extensions/PhotoCapture/CameraIntrinsics.cs

```diff
@@ -51,7 +51,7 @@ public class CameraIntrinsics
         ///     for apps that choose to implement GPU-based distortion compensation instead of
         ///     using UndistortPoint, which uses the CPU to compute the distortion compensation.
         /// </summary>
-        public Matrix4x4 UndistortedProjectionTransform { get; private set; }
+        public Matrix4x4 UndistortedProjectionTransform;
```

Comment:
If you remove properties, you should set default values to these, as they will show up in the unity editor, and uninitialized variables trigger warnings